### PR TITLE
Uganda 2019

### PR DESCRIPTION
**Pull request by adebowaledaniel**
- Trained Uganda 2019 model
-  Updated dataset upsampling in `src/model/data.py`.

### CODE REVIEW

File: src/models/data.py [hunk 1]
```diff
@@ -91,8 +91,8 @@ def __init__(
                     ),
                     ignore_index=True,
                 )
-
             print(f"Upsampling: local crop{arrow}non-crop: {local_crop}{arrow}{local_non_crop}")
+            # TODO: condition for both datasets are equal
```
Comment (ivanzvonkov) [inline src/models/data.py:95]: Please fix!

File: train.py [hunk 2]
```diff
@@ -11,12 +11,12 @@
 from src.pipeline_funcs import train_model
 
 selected_bbox = bboxes["Sudan_Blue_Nile"]
-train_datasets = [datasets[1].dataset]  # [d.dataset for d in datasets]
+# train_datasets = [datasets[1].dataset]  # [d.dataset for d in datasets]
```
Comment (ivanzvonkov) [inline train.py:14]: Looks like CI training time did not increase too much, I am fine with removing this line completely

**Review by ivanzvonkov — CHANGES_REQUESTED** [review]
Requested a couple small changes

**Comment by adebowaledaniel** [comment]
It seems there is an exemption to the `HawaiiAgriculturalLandUse2020` dataset you just added; see the [failed test ](https://github.com/nasaharvest/crop-mask/actions/runs/3632110310/jobs/6127575244#step:15:419)

File: src/models/data.py [hunk 3]
```diff
@@ -75,7 +75,9 @@ def __init__(
             wandb_logger.experiment.config.update(to_log)
 
         if upsample:
-            if local_crop > local_non_crop:
+            if local_crop == local_non_crop:
+                print(f"No upsampling: {local_crop} == {local_non_crop}")
+            elif local_crop > local_non_crop:
```
Comment (ivanzvonkov) [inline src/models/data.py:80]: 👍 👍

**Review by ivanzvonkov — APPROVED** [review]
Looks good, if tests pass, good to merge
